app/export.go

```diff
@@ -16,7 +16,7 @@ import (
 // file.
 func (app *App) ExportAppStateAndValidators(forZeroHeight bool, jailAllowedAddrs, modulesToExport []string) (servertypes.ExportedApp, error) {
 	// as if they could withdraw from the start of the next block
-	ctx := app.NewContext(true, tmproto.Header{Height: app.LastBlockHeight()})
+	ctx := app.NewContext(true /*tmproto.Header{Height: app.LastBlockHeight()}*/)
```

Comment:
Why second argument commented?

Reply:
NewContext takes only one argument which is of type bool.